mmv1/products/cloudfunctions2/terraform.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  CloudFunction:  !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: 'projects/{{project}}/locations/{{region}}/functions/{{cloud_function}}'
+    base_url: projects/{{project}}/locations/{{region}}/functions
+    import_format: ["projects/{{project}}/locations/{{region}}/functions/{{cloud_function}}"]
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudfunctions2_basic"
+        primary_resource_id: "function"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-my-function%s\", context[\"random_suffix\"])"
+        vars:
+          function_name: "my-function"
+          bucket_name: "cloudfunctions-function-example-bucket"
+          zip_path: "path/to/index.zip"
+        test_vars_overrides:
+          zip_path: 'createZIPArchiveForCloudFunctionSource(t, testHTTPTriggerPath)'
+    properties:
+      location: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'region'
```

Comment:
I'd recommend keeping this as `location`, given that's the API name for the field.

Reply:
Updated